### PR TITLE
Added proper semicolons and return comments

### PR DESCRIPTION
**Pull request by JacobGunther12**
Added proper semicolons to the end of each function. Also added a comment in "In the browser" and "In Node" examples for return statement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ In the browser:
 ```html
 <script src="humanize-duration.js"></script>
 <script>
-humanizeDuration(12000)
+humanizeDuration(12000); // '12 seconds'
```
Comment (EvanHahn) [inline README.md]: I like these comment additions—please keep them in your PR.

**Comment by EvanHahn** [comment]
@JacobGunther12 Any updates here?

**Comment by JacobGunther12** [comment]
Oh, I totally forgot about this PR. I'll get working on that.

**Comment by JacobGunther12** [comment]
See #123.

**Pull request closed without merging**